pkg/interceptors/server/server_test.go

```diff
@@ -236,7 +236,7 @@ func createSecret(t *testing.T, noAfter time.Time) (v1.CoreV1Interface, []byte,
 	if _, err := clientSet.Secrets(namespace).Create(ctx, localObject, metav1.CreateOptions{}); err != nil {
 		t.Error(err)
 	}
-	sCert, caCert, err := createCerts(ctx, clientSet, noAfter, logger.Sugar())
+	sCert, caCert, err := createCerts(ctx, clientSet, noAfter, logger.Sugar(), false)
 	return clientSet, sCert, caCert, err
```

Comment:
Don't we need a test for the true scenario also?

Reply:
updated test  to work with  true scenario as well